diesel_cli/Cargo.toml

```diff
@@ -17,15 +17,16 @@ chrono = "0.2.17"
 clap = "2.11"
 diesel = { version = "0.9.0", default-features = false }
 dotenv = "0.8.0"
+diesel_infer_schema = { version = "0.1.0", default-features = false, path = "../diesel_infer_schema" }
```

Comment:
As stated in [here](http://doc.crates.io/specifying-dependencies.html#overriding-with-local-dependencies) this is not possible as long as the crate is not published to crates.io

> Note: using a local configuration to override paths will only work for crates that have been published to crates.io. You cannot use this feature to tell Cargo how to find local unpublished crates.